java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -1410,4 +1414,24 @@ public CompletableFuture<Long> bitpos(
                 };
         return commandManager.submitNewCommand(BitPos, arguments, this::handleLongResponse);
     }
+
+    @Override
+    public CompletableFuture<Long[]> bitfield(
+            @NonNull String key, @NonNull BitFieldOptions.BitFieldSubCommands[] subCommands) {
```

Comment:
You can manuall add static import and
```suggestion
            @NonNull String key, @NonNull BitFieldSubCommands[] subCommands) {
```